### ISSUE TITLE
Startup crash

### ISSUE THREAD

**Issue by Benedikt20** [Haggle1996/RevolutionPack#155]
After a complete reinstall it starts normal, but when I want to play again on the next day it crashes everytime with this crash-report http://paste.atlauncher.com/view/9fe75065

**Comment by DaMachinator** [comment]
Rotarycraft and Electricraft are erroring, can you test with just those two mods?

Pull the mods directly out of your ATLauncher mods folder along with DragonAPI, and the configs if you can find them, and run in a clean Forge instance.

**Comment by DaMachinator** [comment]
If this is a mod bug, we may be out of luck.

Reika is not known for fixing old mod versions.

**Comment by Haggle1996** [comment]
I talked with Benedikt20 this morning on IRC. I'm hoping they supply a crash report without OptiFine.

**Comment by Benedikt20** [comment]
DaMachinator: I  made it so like you said and it isn't crashed but there are a few errors in the log: http://paste.atlauncher.com/view/406a9bed

**Comment by DaMachinator** [comment]
Those errors appear to be "normal" (by which I mean they do not break anything).
So the issue is with another mod.

Did you add any mods?

**Comment by DaMachinator** [comment]
|I talked with Benedikt20 this morning on IRC. I'm hoping they supply a crash report without OptiFine.

It isn't in the mods list; does that mean he is using it as a jarmod?
IIRC you can have Forge load it as a mod.

**Comment by Haggle1996** [comment]
Those errors are normal; For example, the DragonAPI errors occur when certain mods it's compatible with are not present in the pack (for example, we don't have Tails). I don't see OptiFine in the latest log (just the message saying it's not loaded).

I'm going to close this issue, since you said it's working. Please reopen or create a new issue if you crash again at startup.

**Comment by DaMachinator** [comment]
I asked him to run it with *just* the affected mods to see if it was a mod bug or mod interaction,

That was why the latest crash report lacked Optifine.

**Comment by DaMachinator** [comment]
Benedikt20: If you can run the pack without it, please test for a crash with Optifine removed.

**Comment by Benedikt20** [comment]
I found out that the crashes came from a mod called better fps and some extra java arguments for better performance so I tried it with them and without them, and the game crashes only if i use one of them or both, so I removed them and now it isn't crashing anymore. Sorry for bad english, it isn't my mothertongue. Thanks for your help. I love this good support. If I find more problems or bugs I report here.
